### PR TITLE
Implement pushing the app package directly to Google Cloud Storage

### PR DESCRIPTION
**Pull request by markvincze**

### CODE REVIEW

File: cmd/appix/main.go [hunk 1]
```diff
@@ -107,6 +108,7 @@ func makeConfig() config.Config {
 		FirebaseMessagingSenderId: travixFirebaseMessagingSenderId,
 		FirebaseRefreshTokenUrl:   travixFirebaseRefreshTokenUrl,
 		TravixLoggerUrl:           travixLoggerUrl,
+		TravixUploadURL:           travixUploadURL,
```
Comment (discosultan) [inline cmd/appix/main.go]: Could follow existing casing convention: `URL` -> `Url`

**Comment by markvincze** [comment]
@jackTheRipper oh so the actual upload is just a `PUT` HTTP message, so we don't need any of the SDKs, right?

**Comment by jackTheRipper** [comment]
@markvincze : yes

File: firebase-config-helper.js [hunk 2]
```diff
@@ -8,7 +8,8 @@ var config = {
     storageBucket: "fireball-development.appspot.com",
     messagingSenderId: "445151709427",
     certContent: "DummyCertificateContent",
-    keyContent: "DummyKeyContent"
+    keyContent: "DummyKeyContent",
+    uploadUrl: "TRAVIX_UPLOAD_URL=https://us-central1-travix-development.cloudfunctions.net/http"
```
Comment (jackTheRipper) [inline firebase-config-helper.js]: The value of `uploadUrl` should be `https://us-central1-travix-development.cloudfunctions.net/http` without `TRAVIX_UPLOAD_URL`

File: push.go [hunk 3]
```diff
@@ -89,51 +90,86 @@ func push(config config.Config, appPath string, noBrowser bool, wait int, timeou
...
-	if localFrontend {
-		uploadURI, err = replaceURLSchemaAndDomain(uploadURI, "http://localhost:3001")
-	} else if devSettings.DevServerOverride != "" {
-		uploadURI, err = replaceURLSchemaAndDomain(uploadURI, devSettings.DevServerOverride)
+	// request the upload url
+	var uploadObject *SignedUploadURL
+	for attempt := 1; attempt <= config.MaxRetryAttempts; attempt++ {
+		uploadObject, err = RetrieveUploadURL(config.TravixUploadUrl, tb.IdToken, appName, devSettings.SessionID)
+		if err == nil {
+			break
```
Comment (jackTheRipper) [inline push.go]: It could be nice to let the user know that we are re-trying

Comment (nunorfpt) [reply push.go]: Yes, I will add a logging message there

**Pull request merged**